src_text/TCP_connections.txt

```diff
@@ -1,22 +1,22 @@
 
 A method to detect potentially dysfunctional relays that try to disrupt the network by lying that they are connecting to a peer when they are not or that maliciously drop all packets should be considered. Toxcore doesn't currently implement such a system and adding one requires more research and likely also requires extending the protocol.
 
-When TCP connections connects to a relay it will create a new TCP_client instance for that relay. At any time if the TCP_client instance reports that it is disconnected, the TCP relay will be dropped. Once the TCP relay reports that it is connected, TCP_connections will find all the connections that are associated to the relay and announce to the relay that it wants to connect to each of them with routing requests. If the relay reports that the peer for a connection is online, the connection number and relay will be used to send data in that connection with data packets. If the peer isn't reported as online but the relay is associated to a connection, TCP OOB packets will be used to send data instead of data packets. TCP OOB packets are used because in this case the relay most likely has the peer connected however they have not sent a routing request to connect to us.
+When TCP connections connects to a relay it will create a new TCP_client instance for that relay. At any time if the TCP_client instance reports that it has disconnected, the TCP relay will be dropped. Once the TCP relay reports that it is connected, TCP_connections will find all the connections that are associated to the relay and announce to the relay that it wants to connect to each of them with routing requests. If the relay reports that the peer for a connection is online, the connection number and relay will be used to send data in that connection with data packets. If the peer isn't reported as online but the relay is associated to a connection, TCP OOB packets will be used to send data instead of data packets. TCP OOB packets are used because, in this case, the relay most likely has the peer connected but they have not sent a routing request to connect to us.
```

Comment:
>TCP OOB packets

@irungentoo you may want to change that to `TCP OOB (out-of-band)`

>TCP OOB packets are used because, in this case, the relay most likely (…)

perhaps change to ``TCP OOB packets are used in this case since the relay most likely (…)`` ?